charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+helm:
+  # -- Enable the installation of the CRs so FluxCD deploy the Super Agent is deployed. This an advanced/debug flag. It
+  # should be always be true unless you know what you are going. 
+  create: true
+
+  # -- Values related to the Helm repository where to download the super agent's chart.
+  # @default -- See `values.yaml`
+  repository:
+    # -- Sets the interval the repository is going to be updated on the controller.
+    updateInterval: 24h
+    # -- The repository where the super-agent has the chart. This values is meant to be changed only on air-gapped 
+    # environments or for development/testing purposes.
+    url: https://helm-charts.newrelic.com
+
+    # -- A reference to a secret with the keys username and password to authenticate to the repository.
+    # @default -- `nil` (no secret reference) 
+    secretRef:
+      # name: name-of-the-secret
+    # -- secret of type `kubernetes.io/tls` with the standard keys `tls.crt`, `tls.key`, and `ca.crt`
+    # @default -- `nil` (no secret reference) 
+    certSecretRef:
+      # name: name-of-the-secret
+
+  # -- Values related to the super agent's Helm chart release.
+  # @default -- See `values.yaml`
+  release:
+    # -- The Helm chart of the super-agent. This values is meant to be changed only on air-gapped environments or for
+    # development/testing purposes.
+    chart: super-agent-deployment
+    # -- The Helm chart of the super-agent. This values is meant to be changed only on air-gapped environments or for
+    # development/testing purposes.
+    #
+    # TODO: Point renovatebot here.
+    version: "0.0.0-beta"
+
+    # -- Set values to the super agent helm release directly from this `values.yaml` file. Refer to https://fluxcd.io/flux/components/helm/helmreleases/#values-overrides
+    # @default -- no values
+    values:
```

Comment:
would make sense to add the license , perhaps create the secret and configurate it here by default ?

Reply:
I agree, it would be nice to have a working config by default

Reply:
I added it as a TODO on [#d2e49c1](https://github.com/newrelic/helm-charts/pull/1175/commits/d2e49c17d289c548346804a27429633fbe279b38)

We need to make it work downstream before adding it here.